CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 3.6.0 (YYYY-MM-DD)
 
+### Breaking Changes
+
+* [ObjectServer] Read-only is not supported by synced Realm. "SyncConfigration.readOnly()" will always throw `UnsupportedOperationException`.
```

Comment:
I think maybe I could work around this to make the behave the same as current `master` if it is _really not acceptable_ for `master`. But this has to get the same behaviour as the Object Store and other bindings at least from `master-4.0`.